### DISCUSSION TITLE
Version for Ubuntu 22.04 ARM64

### DISCUSSION THREAD

**Discussion by diablo1281** [input-leap/input-leap#2110] [Feature Requests]
### What happened?

I tried to build from source but encounter problems with QT- QT6_FOUND = NO (or something like this). I am out of ideas...

### Version

v3.0.2

### If applicable, where did you install Input Leap from?

Jetson Orin NX 16GB with Jetpack 6.1 running Ubuntu 22.04 on Arm Cortex-A78AE CPU

### What OSes are you seeing the problem on? (Check all that apply)

Linux

### What OS versions are you using?

Jetson Orin NX 16GB with Jetpack 6.1 running Ubuntu 22.04 on Arm Cortex-A78AE CPU

**Comment by charlestephen** [comment]
I second this but a version for Debian ARM64

**Comment by theofficialgman** [comment]
Same, should be easy for Ubuntu 24.04 once GitHub Actions ARM64 Ubuntu 24.04 runners go public in Q1 2025

**Comment by ssshake** [comment]
Yes I came here for the same reason. I am running a triple monitor set up between a windows PC, an x86_64 linux machine and a Jetson Orin. It would be great if I could ditch my physical KVM switch which doesn't work with 3 systems anyway.

An ARM build rather than having to compile would be awesome especially since I think more and more ARM linux workstations will become a thing.

**Comment by sithlord48** [comment]
While i know its not ideal for everyone you can use a flatpak for either input-leap or deskflow as flathub builds both x86_64 as well as aarch64 version.
